### DISCUSSION TITLE
Can I dynamically change the geometry of a featureNode?

### DISCUSSION THREAD

**Discussion by chuyanjuncn** [pelicanmapping/osgearth#1814] [Support Q&A] [answered]
I am using osgEarth::Symbology::Geometry  to construct a feature, then construct the featureNode with the Feature, at last the featureNode is added to mapnode.
Can I dynamically change the geometry of a featureNode?

**Comment by RhythmOfTension** [comment, marked as answer]
You can change geometry and then call `featureNode->dirty()`

**Reply by chuyanjuncn** [reply]
Thanks a lot
